ga4gh/frontend.py

```diff
@@ -105,7 +109,10 @@ def handleException(exception):
 
     error = protocol.GAException()
     error.errorCode = exception.code
-    error.message = exception.message
+    if errorMessage is not None:
```

Comment:
I don't think either of these would work because exception is potentially a different object before and after the two if statements in the middle of `handleException`.  That's why `errorMessage` is saved before it is potentially lost.